---
layout: post
title: ! 'Beware of mis-assembled genomes: still valid today!'
tags:
- De novo assembly
- Genomics
- Paper comments
categories:
- Paper comments
---
<p>I&#8217;ve been recently impressed by Steven Salzberg talk as <a href="http://fellgernon.tumblr.com/post/16468048496/extensive-comments-and-review-about-the-recent-bake-off#.TyAz0OPOzIw">you might have noticed</a>, and browing <a href="http://bioinformatics.igm.jhmi.edu/salzberg/">his home page</a> I stumbled upon his opinion piece (also by James Yorke): <a href="http://cbcb.umd.edu/papers/Mis-assembledGenomesReprint.pdf">Beware of mis-assembled genomes</a>.</p>
<p>It&#8217;s a short note published in 2005, but damn, can anyone deny that it fits perfectly for today&#8217;s state of the art in the de <em>novo</em> genome assembly field? I bet no one will. For instance, it&#8217;s a solid statement to say:</p>
> The source of most mis-assemblies is, as it has always been, repeats.
<p>He didn&#8217;t add a &#8220;will always be&#8221; or &#8220;will be for at least 7 years more&#8221; now that we are in 2012, but it feels like this will be the case until we can get accurate (and cheap) reads that span even the longest repeat. Well, maybe we don&#8217;t need such huge reads as people have been able to find large genome duplications. </p>
<p>And as I said in my <a href="http://fellgernon.tumblr.com/post/16468048496/extensive-comments-and-review-about-the-recent-bake-off#.TyAz0OPOzIw">previous post</a>, I&#8217;m still surprised by how careless the human genome assembly was carried out as they didn&#8217;t track their own steps. I was hoping that wasn&#8217;t the case, but clearly it is:</p>
> Indeed, many of the original assemblies of parts of the human genome were done in the mid- and late-1990s, and are now lost.

<p>I&#8217;m also impressed by how accurate Steven and James&#8217; prediction was when they forsaw that people were going to be misled in judging assembly quality by contigs size without taking into account mis-assemblies. </p>
<p>They also called upon the bioinformatics community to take action in evaluating genome assemblies. Due to the amount of data nowadays, it feels like a inhuman (well, incluster as infeasible by high power clusters :P, well, incomputable is the correct term) task. But with some funding, I bet Salzberg and colleagues could find a way to do so. At least partially. Yet, as with anything, you need motivation, and I&#8217;m note sure they are motivated to clean up the mess. </p>
